sources/packages/api/src/auth/user-groups.enum.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Application groups defined on Keycloak.
+ */
+export enum UserGroups {
+  aestUser = "/aest/user",
```

Comment:
IMO, Enum types generally start with uppercase stating that every enum type is enum object itself.

Reply:
You are right 👍